README.md

```diff
@@ -58,9 +58,13 @@ associated workflow.
 Link triggers are _unique to each installed version of your app_. This means
 that Shortcut URLs will be different across each workspace, as well as between
 [locally run](#running-your-project-locally) and
-[deployed apps](#deploying-your-app). When creating a trigger, you must select
-the Workspace that you'd like to create the trigger in. Each Workspace has a
-development version (denoted by `(dev)`), as well as a deployed version.
+[deployed apps](#deploying-your-app).
+
+When creating a trigger, you must select the workspace and environment that
+you'd like to create the trigger in. Each workspace has a local development
+version (denoted by `(dev)`), as well as a deployed version. Triggers created in
+a local environment will only be available to use when running the application
+locally.
 
 To create a link trigger for the workflow in this template, run the following
```

Comment:
This might also be work for a separate PR, since I can see it starting more discourse due to the different READMEs for apps with multiple vs. single triggers

Reply:
That's a good point. I think we should make note of what you've pointed out and when we have a bit more time, sit down to think it out and tackle it.